constants.go

```diff
@@ -48,6 +48,11 @@ var (
 // UndefAddressString is the string used to represent an empty address when encoded to a string.
 var UndefAddressString = "<empty>"
 
+// MaxUint64StringLength defines the maximum length of `uint64` as a string.
+// FIXME: Can we dynamically get this from something like `math.MaxUint64.Bytes()`
+//  or is this guaranteed to be always the same value?
+const MaxUint64StringLength = 19
```

Comment:
It isn't uint, it is int64: https://github.com/filecoin-project/go-address/pull/31/files#diff-2fdf602be2c46334197afce83240756e39b28d60e1941ea8a07f091c49deaa41R225

But it is guaranteed to be always 19, it comes from simple math of: `log10(2^63-1) = 18.96` which we round up to 19.